### ISSUE TITLE
Feature Requested : RadzenLabel bound to input component should be able to get the text from Input data annotation display attribute 

### ISSUE THREAD

**Issue by paritoshromy** [radzenhq/radzen-blazor#1757]
One of the things that I loved about MVC is the asp-for on label elements. The ability to just provide the display name once as a data annotation and then reuse it at multiple places is really good.

Radzen label text needs to be typed as independent texts. If possible to pick the text from data annotation if custom text attribute is not specified. The label text should come from the component it is tied to. This will improve the development speed for CRUD forms a lot.

This thing can then be replicated to RadzenColumn and the Column header can come as default from the bound property data annotation attribute.

Another cool feature can be adding the required red star symbol if the data annotation has a required property.

**Comment by franklupo** [comment]
Good idea, but in my case the label must go through StringLocalizer.

Best regards

**Comment by paritoshromy** [comment]
Localisation makes it bit tricky but I think that can be manageable through a custom function on the improved label component…need to give it a proper try though.

**Comment by franklupo** [comment]
global localization would be awesome. a method with 2 parameters type and text to translate.

**Comment by akorchev** [comment]
I don't think this can be easily implemented as RadzenLabel doesn't know the model or property. So the DisplayAttribute can't be obtained.
